universal-application-tool-0.0.1/app/views/style/Styles.java

```diff
@@ -100,26 +100,6 @@
   public static final String BG_PINK_700 = "bg-pink-700";
   public static final String BG_PINK_800 = "bg-pink-800";
   public static final String BG_PINK_900 = "bg-pink-900";
-  public static final String BG_TEAL_50 = "bg-teal-50";
```

Comment:
why remove these?

Reply:
They were unused and are no longer being imported by tailwind.config